tutorials/clinical/10_ieeg_localize.py

```diff
@@ -173,42 +219,81 @@ def plot_overlay(image, compare, title, thresh=None):
+if not op.exists(op.join(subjects_dir, 'sample_seeg')):
+    os.symlink(op.join(misc_path, 'seeg', 'sample_seeg'),
+               op.join(subjects_dir, 'sample_seeg'))
+
+# estimate head->mri transform
+subj_trans = mne.coreg.estimate_head_mri_t('sample_seeg', subjects_dir)
+
 # %%
 # Marking the Location of Each Electrode Contact
 # ==============================================
 #
 # Now, the CT and the MR are in the same space, so when you are looking at a
 # point in CT space, it is the same point in MR space. So now everything is
 # ready to determine the location of each electrode contact in the
-# individual subject's anatomical space (T1-space). To do this, can make
-# list of ``TkReg RAS`` points from the lower panel in freeview or use the
-# mne graphical user interface (coming soon). The electrode locations will then
-# be in the ``surface RAS`` coordinate frame, which is helpful because that is
-# the coordinate frame that all the surface and image files that freesurfer
-# outputs are in, see :ref:`tut-freesurfer-mne`.
+# individual subject's anatomical space (T1-space). To do this, we can use the
+# MNE intracranial electrode location graphical user interface. The electrode
+# locations.
 #
-# The electrode contact locations could be determined using freeview by
-# clicking through and noting each contact position in the interface launched
-# by the following command:
+# .. note: The most useful coordinate frame for intracranial electrodes is
+#          generally the ``surface RAS`` coordinate frame because that is
+#          the coordinate frame that all the surface and image files that
+#          Freesurfer outputs are in, see :ref:`tut-freesurfer-mne`. These are
+#          useful for finding the brain structures nearby each contact and
+#          plotting the results.
 #
-# .. code-block:: bash
+# To operate the GUI, there are two options:
 #
-#     $ freeview $MISC_PATH/seeg/sample_seeg_T1.mgz \
-#           $MISC_PATH/seeg/sample_seeg_CT.mgz
-
-# %%
-# Now, we'll make a montage with the channels that we've found in the
-# previous step.
+# 1) Manually
 #
-# .. note:: MNE represents data in the "head" space internally
+#   - Click in each image to naviate to each electrode contact
+#   - Select the contact name in the right panel
+#   - Press "Mark" to associate that position with that contact
+#
+#   .. note: Using the scroll or +/- arrow keys you can zoom in and
+#            out, and the up/down, left/right and page up/page down keys
+#            allow you to move one slice in any direction. This information
+#            is available in the help menu, accessible by pressing "h".
+#
+# 2) Using "Auto-Find"
+#
+#   - Find a typical electrode contact err toward the largest type used
+#   - Press "Mark" to put a marker in that location
+#   - Adjust the radius slider so that it is just bigger than the contact
+#   - Adjust the CT threshold slider so that the contacts have black
+#     space between them
```

Comment:
```suggestion
#   - Find a typical electrode contact err toward the largest type used.
#   - Press "Mark" to put a marker in that location.
#   - Adjust the radius slider so that it is just bigger than the contact.
#   - Adjust the CT threshold slider so that the contacts have black
#     space between them.
```